packages/server/src/unstable-core-do-not-import/stream/sse.ts

```diff
@@ -32,7 +32,7 @@ export interface PingOptions {
 export interface SSEStreamProducerOptions<TValue = unknown> {
   serialize?: Serialize;
   data: AsyncIterable<TValue>;
-  abortCtrl: AbortController;
```

Comment:
This shouldn't be needed anymore

We have this: https://github.com/trpc/trpc/blob/8b37fed52bc53acea6df9bb0896a8cdc89f8e39c/packages/server/src/adapters/node-http/writeResponse.ts#L47-L49